pkg/apis/contrail/v1alpha1/vrouter_types.go

```diff
@@ -50,32 +50,33 @@ type VrouterStatus struct {
 // VrouterSpec is the Spec for the vrouter API.
 // +k8s:openapi-gen=true
 type VrouterSpec struct {
-	CommonConfiguration       PodConfiguration            `json:"commonConfiguration,omitempty"`
-	ServiceConfiguration      VrouterServiceConfiguration `json:"serviceConfiguration"`
-	EnvVariablesConfiguration map[string]string           `json:"envVariablesConfiguration,omitempty"`
+	CommonConfiguration  PodConfiguration            `json:"commonConfiguration,omitempty"`
+	ServiceConfiguration VrouterServiceConfiguration `json:"serviceConfiguration"`
 }
 
 // VrouterServiceConfiguration defines all vRouter service configuration
 // +k8s:openapi-gen=true
 type VrouterServiceConfiguration struct {
 	VrouterConfiguration      `json:",inline"`
 	VrouterNodesConfiguration `json:",inline"`
+	EnvVariablesConfiguration map[string]string `json:"envVariablesConfiguration,omitempty"`
```

Comment:
You added it to VrouterConfiguration struct already, so you do not need to add it here again